templates/index.html

```diff
@@ -40,7 +40,7 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
         <h2>Straightforward installation</h2>
         <h3 class="p-heading--4">Fully functional OpenStack in 2 commands and ~20 minutes.</h3>
       </div>
-      <pre class="p-heading--two"><code>$ sudo <span class="typer" id="main" data-words="snap install microstack --beta --devmode,microstack init --auto --control" data-delay="50" data-deleteDelay="5000" data-colors="#111111"></span></code></pre>
+      <div class="row-cli__commands" style="margin-bottom: 1rem;">$ sudo<span class="row-cli__commands-list typer" id="main" data-words="snap install microstack --beta --devmode,microstack init --auto --control" data-delay="50" data-deleteDelay="5000" data-colors="#111111"></span></div>
```

Comment:
Could you not add the margin bottom style to the `row-cli__commands` class?

Reply:
Fixed! :)

Reply:
Fixed 👍